pages/bookmarks.jsx

```diff
@@ -37,7 +37,7 @@ export default React.createClass({
     return (<p>Only Mozilla staff can login now as we test this new platform. Meanwhile, your favorited projects can still be remembered on this device.</p>);
   },
   getAnonymousContent() {
-    return (<p>Are you Mozilla Staff? <button className="btn btn-link inline-link" onClick={this.handleSignInBtnClick}>Sign in</button>.</p>);
+    return (<p>Are you Mozilla Staff? <a href="javascript:void(0)" onClick={this.handleSignInBtnClick}>Sign in</a>.</p>);
```

Comment:
This is not a good solution to the problem of a button vs. a real link - `href="javascript:void(0)"` has all kinds of problems; CSP will disallow `javascript:` for instance, and an `<a>` in HTML5 **has** to point somewhere. It can't be used as a placeholder for a thing that pretends to be a link anymore: if it's an `<a>` it *has* to point to a URI (either a `#fragmentid` or an external URL. Empty href, or `javascript:` or just `#` are not correct use of the element anymore)

So instead the solution is to make this a *true* link, so that the link looks like:
```
<a href={ this.createLoginLink }>Sign in</a>
```
so no onClick but instead a function that generates the URL that users need to be redirected too, currently generated somewhere inside of `handleSignInBtnClick` (see https://github.com/Sudip7/network-pulse/blob/4fbbb73ff6956e327a930ebc5dcd11e2ab6899b5/pages/bookmarks.jsx#L19 where `user.login` happens. Instead of that function doing the redirect, that functionality needs to now become a URL that the `<a>` handles instead.